CONTRIBUTING.md

```diff
@@ -17,6 +17,14 @@ The point of lspconfig is to provide the minimal configuration necessary for a s
 
 ## Adding a server to lspconfig
 
+New configs must meet these criteria:
+
+- GitHub Stars: The server repository should have at least 100 stars, or some other evidence (such as vscode marketplace downloads) that the LSP server is reasonably popular and is not spam/quasi-marketing/vanity projects.
+- VS Code Market: The server should have a reasonable amount of downloads in the VS Code marketplace.
```

Comment:
making vscode marketplace a hard requirement is going to get complaints/confusion if the server isn't on the vscode marketplace.

```suggestion
- Provide some reference or evidence that the language targeted by the LSP server has an active user base.
```